samples/bluetooth/mesh/sensor_client/src/model_handler.c

```diff
@@ -66,9 +73,8 @@ static void sensor_cli_desc_cb(struct bt_mesh_sensor_cli *cli, struct bt_mesh_ms
 			       const struct bt_mesh_sensor_info *sensor)
 {
 	printk("Descriptor of sensor with ID 0x%04x:\n", sensor->id);
-	printk("\ttolerance: { positive: %s",
-	       bt_mesh_sensor_ch_str(&sensor->descriptor.tolerance.positive));
-	printk(" negative: %s }\n", bt_mesh_sensor_ch_str(&sensor->descriptor.tolerance.negative));
+	printk("\ttolerance: { positive: %d", sensor->descriptor.tolerance.positive);
+	printk(" negative: %d }\n", sensor->descriptor.tolerance.negative);
```

Comment:
Why is this not put into one printk, instead of 2?

Reply:
Good question. I just modified the existing lines above, but since we don't call `bt_mesh_sensor_ch_str` I can join them, true.